server/schedule/operator/operator.go

```diff
@@ -101,6 +101,7 @@ func (s StoreInfluence) ResourceSize(kind core.ResourceKind) int64 {
 // OpStep describes the basic scheduling steps that can not be subdivided.
 type OpStep interface {
 	fmt.Stringer
+	ConsumeConfVer() int
```

Comment:
em, I think `Consume` seems hard to understand. Actually, the step does not 'consume' the config version, but trigger the update of config version. How about something like `ExpectConfVarChange() bool`?

Reply:
You mean `ExpectConfVerChange() bool`? OK, I also would agree.